docs/api/app.md

```diff
@@ -760,6 +760,23 @@ Disables hardware acceleration for current app.
 
 This method can only be called before app is ready.
 
+### `app.getAppMemoryInfo()`
+
+Returns `Object[]`:
+
+* `pid` Integer - The process id for which memory info is collected for
```

Comment:
Does this allow to link to a specific `web-contents`? Can I use `webContents.fromId(pid)`?